src/annotator/anchoring/pdf.js

```diff
@@ -470,8 +469,8 @@ export function describe(root, range, options = {}) {
     );
 
     const quoteRange = document.createRange();
-    quoteRange.setStartBefore(startRange.start);
-    quoteRange.setEndAfter(endRange.end);
+    quoteRange.setStartBefore(startRange?.start);
+    quoteRange.setEndAfter(endRange?.end);
```

Comment:
What will happen here if `startRange` is in fact nullish? Are we in fact just assuming that it is not?

Reply:
So limit() can return null and it was never a case being handled here. Its still not handled per se, but we could perhaps throw an error. I'm just not sure what such a case would actually look like in the wild so I did not want to make any assumptions.